controls/V-72027.rb

```diff
@@ -49,27 +43,26 @@
 -rwxr----- 1 smithj smithj 193 Mar  5 17:06 file2
 -rw-r-x--- 1 smithj smithj 231 Mar  5 17:06 file3
 
-If any files are found with a mode more permissive than \"0750\", this is a finding."
-  tag "fix": "Set the mode on files and directories in the local interactive user
-home directory with the following command:
+If any files are found with a mode more permissive than \"0750\", this is a
+finding."
+  tag "fix": "Set the mode on files and directories in the local interactive
+user home directory with the following command:
 
 Note: The example will be for the user smithj, who has a home directory of
 \"/home/smithj\" and is a member of the users group.
 
 # chmod 0750 /home/smithj/<file>"
+  tag "fix_id": "F-78379r1_fix"
 
-  # Assumption - users' home directories created in "home"
-  # Allows for mode 750 or less permissive
-  home_dirs = command('ls -d /home/*').stdout.split("\n")
-  home_dirs.each do |home|
-    home_files = command("find #{home} -xdev ! -name '.*' -type f -perm /027").stdout.split("\n")
-    home_files.each do |filename|
-      describe file(filename) do
-        it { should_not be_writable.by('group') }
-        it { should_not be_executable.by('others') }
-        it { should_not be_writable.by('others') }
-        it { should_not be_readable.by('others') }
-      end
-    end
+  IGNORE_SHELLS = NON_INTERACTIVE_SHELLS.join('|')
```

Comment:
Global variables can cause issues during test runs. Please convert to a local variable.